pkgs/build-support/rust/default-crate-overrides.nix

```diff
@@ -14,7 +14,7 @@ in
     buildInputs = [ openssl zlib curl ]
       ++ stdenv.lib.optionals stdenv.isDarwin [ CoreFoundation libiconv ];
     # TODO: buildRustCrate seems to use incorrect default inference
-    crateBin = [ {  name = "cargo"; path = "src/bin/cargo.rs"; } ];
+    crateBin = [ {  name = "cargo"; path = "src/bin/cargo/main.rs"; } ];
```

Comment:
You can remove the `path` attribute. Since fc5e5950031d8af57f8b5b9e55187f3e4cb4f063 we are doing the right thing(tm) within `buildRustCrate`.

Reply:
You are right, fixed